HDF5Examples/config/cmake/grepTest.cmake

```diff
@@ -23,7 +24,7 @@ if (NOT TEST_OUTPUT)
   message (FATAL_ERROR "Require TEST_OUTPUT to be defined")
 endif ()
 if (NOT TEST_FILTER)
-  message (STATUS "Optional TEST_FILTER to be defined")
+  message (VERBOSE "Optional TEST_FILTER to be defined")
```

Comment:
The meaning of this message seems unclear.  Following the if condition and with no action to define TEST_FILTER afterwards, would "Optional TEST_FILTER is not defined" be more accurate?